src/uistash.css

```diff
@@ -13,20 +17,20 @@
   border-radius: 4px;
 }
 .alert--success {
-  background-color: hsl(120, 100%, 80%);
-  color: hsl(120, 100%, 20%);
+  background-color: #99FF99;
+  color: #006600;
 }
 .alert--info {
-  background-color: hsl(240, 100%, 80%);
-  color: hsl(240, 100%, 20%);
+  background-color: #9999FF;
+  color: #000066;
 }
 .alert--warning {
-  background-color: hsl(60, 100%, 80%);
-  color: hsl(60, 100%, 20%);
+  background-color: #FFFF99;
+  color:#666600;
 }
 .alert--error {
-  background-color: hsl(0, 100%, 80%);
-  color: hsl(0, 100%, 20%);
+  background-color: #FF9999;
+  color: #660000;
```

Comment:
Good approach! You can also use plugins which convert hsl to rgba/hex in final build. But for now, LGTM!